streamer/src/nonblocking/quic.rs

```diff
@@ -153,8 +153,13 @@ async fn setup_connection(
                 }
                 connection.set_max_concurrent_uni_streams(
                     VarInt::from_u64(
-                        ((stake as f64 / total_stake as f64) * QUIC_TOTAL_STAKED_CONCURRENT_STREAMS)
-                            as u64,
+                        u64::try_from(
```

Comment:
Why is u128 better than f64 for this case?

Reply:
integers are indisputably better than floats in all cases

Reply:
I guess it is faster in multiplication/division arithmetics?

Reply:
less concerned about speed than (predictability of) loss of precision